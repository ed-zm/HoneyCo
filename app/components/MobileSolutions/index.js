import React from 'react'
import { Col, Row, Container } from 'reactstrap'
import styled from 'styled-components'
import { HoneyCoLightGray, HoneyCoDarkGray, HoneyCoYellow } from '../../constants'
import Mobile1 from './static/mobile1.png'
import Mobile2 from './static/mobile2.png'
import Mobile3 from './static/mobile3.png'

const StyledImage = styled.img`
  width: 100%;
`
const StyledTitle = styled.span`
  display: block;
  font-size: 70px;
  text-align: center;
  width: 100%;
  margin-bottom: .5em;
`
const StyledText = styled.span`
  display: block;
  font-size: 30px;
  text-align: center;
  width: 100%;
  
`
const StyledRow = styled(Row)`
  background-color: ${HoneyCoLightGray};
  color: ${HoneyCoDarkGray};
  padding: 1em;
  padding-bottom: 8em;
  margin: 0px;
`
const StyledButton = styled.button`
  font-family: Oswald;
  font-size: 26px;
  background-color: ${HoneyCoYellow};
  padding: .2em;
  border-radius: 5px;
  border: none;
  padding: .3em 1.5em;
  text-align: center;
  margin-top: 10px;
`
const StyledButtonContainer = styled.div`
  text-align: center;
`

const MobileSolutions = () =>
    <StyledRow>
      <Container>
        <Row>
          <StyledTitle>Diferent needs. One Solution.</StyledTitle>
          <Col xl = {4} lg={4} md={12} sm ={12} xs={12} >
            <StyledImage src = {Mobile1}/>
            <StyledText>I want to stay in my home.</StyledText>
            <StyledButtonContainer>
              <StyledButton>Show me how</StyledButton>
            </StyledButtonContainer>
          </Col>
          <Col xl = {4} lg={4} md={12} sm ={12} xs={12} >
            <StyledImage src = {Mobile2}/>
            <StyledText>I want peace of mind.</StyledText>
            <StyledButtonContainer>
              <StyledButton>Show me how</StyledButton>
            </StyledButtonContainer>
          </Col>
          <Col xl = {4} lg={4} md={12} sm ={12} xs={12} >
            <StyledImage src = {Mobile3}/>
            <StyledText>I want to be different</StyledText>
            <StyledButtonContainer>
              <StyledButton>Show me how</StyledButton>
            </StyledButtonContainer>
          </Col>
        </Row>
      </Container>
    </StyledRow>
export default MobileSolutions
