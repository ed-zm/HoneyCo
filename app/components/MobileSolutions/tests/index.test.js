import React from 'react'
import { shallow } from 'enzyme'
import MobileSolutions from '../index'

describe('<MobileSolutions />', () => {
  it('It should render the MobileSolutions Component', () => {
    const wrapper = shallow(<MobileSolutions/>)
    expect(wrapper).toMatchSnapshot()
  })
})
