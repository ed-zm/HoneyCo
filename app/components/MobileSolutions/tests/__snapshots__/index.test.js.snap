// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MobileSolutions /> It should render the MobileSolutions Component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Styled(bt)>
    <gt
        tag="div"
    >
        <bt
            tag="div"
        >
            <styled.span>
                Diferent needs. One Solution.
            </styled.span>
            <Ot
                lg={4}
                md={12}
                sm={12}
                tag="div"
                widths={
                    Array [
                        "xs",
                        "sm",
                        "md",
                        "lg",
                        "xl",
                      ]
                }
                xl={4}
                xs={12}
            >
                <styled.img
                    src="IMAGE_MOCK"
                />
                <styled.span>
                    I want to stay in my home.
                </styled.span>
                <styled.div>
                    <styled.button>
                        Show me how
                    </styled.button>
                </styled.div>
            </Ot>
            <Ot
                lg={4}
                md={12}
                sm={12}
                tag="div"
                widths={
                    Array [
                        "xs",
                        "sm",
                        "md",
                        "lg",
                        "xl",
                      ]
                }
                xl={4}
                xs={12}
            >
                <styled.img
                    src="IMAGE_MOCK"
                />
                <styled.span>
                    I want peace of mind.
                </styled.span>
                <styled.div>
                    <styled.button>
                        Show me how
                    </styled.button>
                </styled.div>
            </Ot>
            <Ot
                lg={4}
                md={12}
                sm={12}
                tag="div"
                widths={
                    Array [
                        "xs",
                        "sm",
                        "md",
                        "lg",
                        "xl",
                      ]
                }
                xl={4}
                xs={12}
            >
                <styled.img
                    src="IMAGE_MOCK"
                />
                <styled.span>
                    I want to be different
                </styled.span>
                <styled.div>
                    <styled.button>
                        Show me how
                    </styled.button>
                </styled.div>
            </Ot>
        </bt>
    </gt>
</Styled(bt)>,
  "nodes": Array [
    <Styled(bt)>
      <gt
            tag="div"
      >
            <bt
                  tag="div"
            >
                  <styled.span>
                        Diferent needs. One Solution.
                  </styled.span>
                  <Ot
                        lg={4}
                        md={12}
                        sm={12}
                        tag="div"
                        widths={
                              Array [
                                    "xs",
                                    "sm",
                                    "md",
                                    "lg",
                                    "xl",
                                  ]
                        }
                        xl={4}
                        xs={12}
                  >
                        <styled.img
                              src="IMAGE_MOCK"
                        />
                        <styled.span>
                              I want to stay in my home.
                        </styled.span>
                        <styled.div>
                              <styled.button>
                                    Show me how
                              </styled.button>
                        </styled.div>
                  </Ot>
                  <Ot
                        lg={4}
                        md={12}
                        sm={12}
                        tag="div"
                        widths={
                              Array [
                                    "xs",
                                    "sm",
                                    "md",
                                    "lg",
                                    "xl",
                                  ]
                        }
                        xl={4}
                        xs={12}
                  >
                        <styled.img
                              src="IMAGE_MOCK"
                        />
                        <styled.span>
                              I want peace of mind.
                        </styled.span>
                        <styled.div>
                              <styled.button>
                                    Show me how
                              </styled.button>
                        </styled.div>
                  </Ot>
                  <Ot
                        lg={4}
                        md={12}
                        sm={12}
                        tag="div"
                        widths={
                              Array [
                                    "xs",
                                    "sm",
                                    "md",
                                    "lg",
                                    "xl",
                                  ]
                        }
                        xl={4}
                        xs={12}
                  >
                        <styled.img
                              src="IMAGE_MOCK"
                        />
                        <styled.span>
                              I want to be different
                        </styled.span>
                        <styled.div>
                              <styled.button>
                                    Show me how
                              </styled.button>
                        </styled.div>
                  </Ot>
            </bt>
      </gt>
</Styled(bt)>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <MobileSolutions />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Styled(bt)>
          <gt
                    tag="div"
          >
                    <bt
                              tag="div"
                    >
                              <styled.span>
                                        Diferent needs. One Solution.
                              </styled.span>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want to stay in my home.
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want peace of mind.
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want to be different
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                    </bt>
          </gt>
</Styled(bt)>,
        "_debugID": 2,
        "_renderedOutput": <Styled(bt)>
          <gt
                    tag="div"
          >
                    <bt
                              tag="div"
                    >
                              <styled.span>
                                        Diferent needs. One Solution.
                              </styled.span>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want to stay in my home.
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want peace of mind.
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                              <Ot
                                        lg={4}
                                        md={12}
                                        sm={12}
                                        tag="div"
                                        widths={
                                                  Array [
                                                            "xs",
                                                            "sm",
                                                            "md",
                                                            "lg",
                                                            "xl",
                                                          ]
                                        }
                                        xl={4}
                                        xs={12}
                              >
                                        <styled.img
                                                  src="IMAGE_MOCK"
                                        />
                                        <styled.span>
                                                  I want to be different
                                        </styled.span>
                                        <styled.div>
                                                  <styled.button>
                                                            Show me how
                                                  </styled.button>
                                        </styled.div>
                              </Ot>
                    </bt>
          </gt>
</Styled(bt)>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <MobileSolutions />,
}
`;
